docs/src/layout/grid/grid.jsx

```diff
@@ -676,8 +676,6 @@ export default class CollectionsGrid extends React.Component {
 
         return (
             <Main className="grid">
-                <TitleBar title="Grid" />
```

Comment:
`<TitleBar />` was removed in here but not in line 5 when importing from `react-cm-ui`.

Reply:
Fixed.